### ISSUE TITLE
Mathematical priority

### ISSUE THREAD

**Issue by Majed6** [HabitRPG/habitica-zapier#11]
I would love to have the priority field take a mathematical equation and calculate what the priority / difficulty should be. 

For example when I add a to-do to read an article via the integration, I would like it to calculate the following equation words count ( provided by another integration) / 1000 in the priority field. 

IMHO, this adds a much more clever way of determining the priority of a task.

**Comment by paglias** [comment]
Sounds good to me, @Alys what do you think?

**Comment by Alys** [comment]
Yes, definitely. It's referring to some code from [this PR](https://github.com/HabitRPG/habitica-zapier/pull/3) that I removed in [this commit](https://github.com/HabitRPG/habitica-zapier/commit/a7f8b57ad2b5ef237b58ceac5d28c4dfab4aec3e) for the reasons described there.

Now that I'm a lot more familiar with zapier, I'm happy to have that added. It's definitely an exciting feature!

@Majed6 Is that still the code you want for this feature? If so, I'll merge it from that commit in a day or two. Otherwise, you're welcome to do a new PR for this if you wish.

**Comment by Majed6** [comment]
I'll do another PR with multiple Test cases.

Thank you
